keanu-project/src/main/java/io/improbable/keanu/util/ProgressBar.java

```diff
@@ -12,7 +12,7 @@
 
 public class ProgressBar {
 
-    public static PrintStream defaultPrintStream = System.out;
+    private static final PrintStream DEFAULT_PRINT_STREAM = System.out;
```

Comment:
This one is actually intentional. It's meant to be a global switch to a specific printstream for the progress bar.

Reply:
I made it private, non-final and added a getter